doc/03_reference/cs_registers.rst

```diff
@@ -140,9 +140,77 @@ Machine ISA Register (misa)
 
 CSR Address: ``0x301``
 
+Reset Value: ``0x4010_1104``
+
 ``misa`` is a WARL register which describes the ISA supported by the hart.
 On Ibex, ``misa`` is hard-wired, i.e. it will remain unchanged after any write.
 
+Detailed:
+
++-------------+------------+------------------------------------------------------------------------+
+| **Bit #**   | **Mode**   | **Description**                                                        |
++=============+============+========================================================================+
+| 31:30       | RO  (0x01) | **MXL** (Machine XLEN)                                                 |
++-------------+------------+------------------------------------------------------------------------+
+| 29:26       | RO   (0x0) | (Reserved)                                                             |
++-------------+------------+------------------------------------------------------------------------+
+| 25          | RO   (0x0) | **Z** (Reserved)                                                       |
++-------------+------------+------------------------------------------------------------------------+
+| 24          | RO   (0x0) | **Y** (Reserved)                                                       |
++-------------+------------+------------------------------------------------------------------------+
+| 23          | RO   (0x0) | **X** (Non-standard extensions present)                                |
++-------------+------------+------------------------------------------------------------------------+
+| 22          | RO   (0x0) | **W** (Reserved)                                                       |
++-------------+------------+------------------------------------------------------------------------+
+| 21          | RO   (0x0) | **V** (Tentatively reserved for Vector extension)                      |
++-------------+------------+------------------------------------------------------------------------+
+| 20          | RO   (0x1) | **U** (User mode implemented)                                          |
```

Comment:
The user mode has been removed

Reply:
Really?  The [CV32E20 Specification](https://docs.openhwgroup.org/projects/cve2-user-manual/en/latest/01_specification/index.html#operating-modes-privilege-levels) does not explicitly state this and the requirement [CSR-10](https://docs.openhwgroup.org/projects/cve2-user-manual/en/latest/01_specification/index.html#operating-modes-privilege-levels) explicitly mentions User Mode.

Reply:
indeed does not look like - I was getting confused with the old zero-riscy where this was a parameter, which it was false by default